cves/kernel/CVE-2016-3135.yml

```diff
@@ -55,7 +55,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```

Comment:
How does this relate to the system that the code was in? i.e. what does this affect in the system in which the vulnerability was found.

Reply:
Added a description of how it can affect many parts of the system.